.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     branches:
-      - main 
+      - main
```

Comment:
Some non-relevant changes were triggered by formatting in VScode but looks like they are valid.